### ISSUE TITLE
Cannot see any messages, only "Keep your phone connected "

### ISSUE THREAD

**Issue by bammerlaan** [92lleo/WhatsappWebToGo#73]
![Screenshot_20220128-135704_WhatsApp Web To Go](https://user-images.githubusercontent.com/3306437/151551104-647ebed4-512e-401a-9a03-babca4604ee5.jpg)
As in the attached screenshot, when I connect my tablet to my phone, I only see this and no messages. 

I tried reconnecting the device, no dice. 

The tablet is a Samsung SM-T590

**Comment by keithnet** [comment]
Same. 

Samsung Tab S7+

**Comment by 92lleo** [comment]
Hi @bammerlaan & @keithnet 
you are on 1.7.2, right? 

- Can you please install an older version and check if it works then?
- Are you using the "normal" wa web or the multi device beta?
- are you both using a vpn connection?

**Comment by 92lleo** [comment]
Thanks for the quick feedback. Does 1.7.1 also work? would be good to know to narrow down the issue

**Comment by keithnet** [comment]
Yes, 1.7.1 works too. Was worried it would have previously reported display issue (#69) . It doesn't.

**Comment by DaStivi** [comment]
i've got an Xiaomi Pad5,
with 1.7.2 i've only like 5mm of the contac list visible... not even seeing half of the profile pictures...

with 1.7.1 i'll see the contact list, but some parts of the chat windows is cut off on the right side...

all this on portrait mode!

in landscape mode everthing fits, in either versions!

using whatsapp web in edge browser (desktop view), portait mode also works fine, whats very different the "zoom" level of the webpage is smaller, so contact list and chat pain, have smaller letters etc... gooing into landscape mode the zoom level gets bigger, letter size increases...

**Comment by witneyman** [comment]
> Can confirm that switching to 1.7.1 (from 1.7.2) seems to fix the issue!
> 
> I'm not on the multi-device beta, nor am I using a vpn.

Also can confirm that switching back to 1.7.1 fixes the issue on my Lenovo Tab M10

**Comment by voidplayer** [comment]
I dont know if im having the same issue but its similar

Im using multidevice

Its very similar screen, but mine said to keep my computer connected instead and a button to log out

Downgrading to 1.7.1 seems to have fixed it
